### ISSUE TITLE
Accessibility Testing for Medications on VA.gov

### ISSUE THREAD

**Issue by BobbyBaileyRB** [department-of-veterans-affairs/va.gov-team#64941]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

BobbyBaileyRB

### Use of color and color contrast checks

- [ ] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### Dropdown button
URL: [Medications](https://staging.va.gov/my-health/medications)
The medications page has a dropdown of "Print and download this list". As of now, the active state fails color contrast at a ratio of 4.05:1 normal text.
- .print-download button:active foreground at a hex of 0071bb;
- .print-download button:active background at a dce4ef;

### Who completed the axe scans?

BobbyBaileyRB

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

0 automatic issues found using Axe on Google Chrome, Mac.

### Who completed the content zoom and reflow test?

BobbyBaileyRB

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### Who completed the keyboard navigation test?

BobbyBaileyRB

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### Email address **(note, might be fixed before staging review)**
URL: [About-medications](https://staging.va.gov/my-health/about-medications)
On the About Medications page, the email address "vamhvfeedback@va.gov" has no response to the expected keys and the link does not go anywhere

### Dropdown button focus
URL: [Medications](https://staging.va.gov/my-health/medications)
On the Medications, there is a dropdown button labeled "Print or download this list". The focus state does not match other keyboard focus states, such as having a yellow border around the component when it receives focus. However, the button does change to a "darker" blue background but that contrast fails normal text (please review section 3. Use of color and color contrast)

### Dropdown component. No cursor pointer
URL: [About-medications](https://staging.va.gov/my-health/about-medications)
The component found in the VA Design System for selecting an item from a dropdown has no cursor pointer change. The sorting component for medications is using this component. Is this the intended/expected functionality?
This button is also at 40px in height, a little under the recommended 44 by 44 pixels or larger.

### Show medications in this order. Sort button
URL: [About-medications](https://staging.va.gov/my-health/about-medications)
This button has a height of 36px and is a little under the recommended 44 by 44 pixels or larger. The sort button has margin-top: 8px and margin-bottom: 8px. Yet those margins are not clickable.

### Accordions
URL: [About-medications](https://staging.va.gov/my-health/about-medications)
The accordions "Expand all +" button is visually coded as "Expand all +". The code uses Aria-label="Expand all accordions". This might be great for screen readers, however, a little confusing for voice control people. 
As of now, a person cannot say "click, expand all". They must say "Click, expand all accordions" or use the numbers grid. This is also within the VA Design System.
This was tested on Mac using Voice control.

### Divs
URL: [Medications](https://staging.va.gov/my-health/medications)
Recommendation of using p-tags instead of div-tags for text within the cards found on the list-view page (medications page).
